Reenable fog toggle unless server restricts fog distance (#14634)
minetest/minetest: doc/lua_api.md

                `"default"` uses the classic Minetest sun and moon tinting.
                Will use tonemaps, if set to `"default"`. (default: `"default"`)
        * `fog`: A table with following optional fields:

>             * `fog_distance`: integer, set an upper bound the client's viewing_range.

Reviewer: ```suggestion
            * `fog_distance`: integer, set an upper bound for the client's viewing_range.
```

               Any value >= 0 sets the desired upper bound for the client's viewing_range,

>                disables range_all and disallows disabling fog (F3 key).

Reviewer 2: ```suggestion
               disables range_all and ignores the setting `keymap_toggle_fog`.
```

Because the key can be changed. Alternatively

```suggestion
               disables range_all and disallows disabling fog by key bindings.
```

Author: how about this? I wanted to mention the key so that the reader knows which function this is referring to.
```suggestion
               disables range_all and disallows disabling fog (default: F3 key).
```

Reviewer 2: That has potential to go out-of-sync in case of a default change but I don't think that is likely to happen. Thus, fine by me.
:+1:
